backend/postgres/db/migrations/000001_initial.up.sql

```diff
@@ -0,0 +1,73 @@
+DROP TABLE IF EXISTS instances;
```

Comment:
Given that this is a new backend, we should merge the migrations into a single `initial` one